script.js

```diff
@@ -0,0 +1,116 @@
+class Collection {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+let book = [];
+
+// Book displayed on the page
+
+function init() {
+  for (let i = 0; i < book.length; i += 1) {
+    const section = document.querySelector('.bookSection');
+    const div = document.createElement('div');
+    div.className = ('bookList');
+    const title = document.createElement('p');
+    title.textContent = (book[i].title);
+    title.className = ('classTitle');
+    div.appendChild(title);
+    const author = document.createElement('p');
+    author.textContent = (book[i].author);
+    author.className = ('classAuthor');
+    div.appendChild(author);
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = ('Remove');
+    removeBtn.className = ('remove');
+
+    /* eslint-disable no-loop-func */
+    removeBtn.addEventListener('click', () => {
+      div.style.display = ('none');
+      for (let i = 0; i < book.length; i += 1) {
+        if (book[i].title === title.textContent) {
+          book.splice(i, 1);
+          localStorage.setItem('list', JSON.stringify(book));
```

Comment:
- [OPTIONAL] 

- [ ] Consider using array methods that don't mutate the array, e.g. concat, filter, slice instead of push, pop, splice in all your codes. 
- [ ] I added 3 books with the same title and author and when I removed one and refresh the page, two books were removed. To make sure this is resolved properly, consider using a unique `id` property for each book since it would make it easier to remove books with duplicate values 😄.

Reply:
Thank you for your review! it's very informative :grinning: